From a1e68ab5305ff6ecbd7544f82e82781f378ceab5 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Mon, 10 Apr 2017 09:56:32 +0800
Subject: [PATCH 153/347] arm: keystone: select GPMC by default

This patch comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

original one From 3aaba44c364a7e310bda5fa5e71aa1163f935706

Some Keystone devices include a GPMC that reuses the IP from OMAP
devices. Therefore, allow the GPMC driver to be useable for both
Keystone and OMAP devices

Signed-off-by: Roger Quadros <rogerq@ti.com>
Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
Signed-off-by: Franklin S Cooper Jr <fcooper@ti.com>
[nsekhar@ti.com: user prompt, default n dropped.
		 Add select on ARCH_KEYSTONE]
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
Signed-off-by: Kishon Vijay Abraham I <kishon@ti.com>
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-keystone/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-keystone/Kconfig b/arch/arm/mach-keystone/Kconfig
index db69bbc..442f393 100644
--- a/arch/arm/mach-keystone/Kconfig
+++ b/arch/arm/mach-keystone/Kconfig
@@ -10,6 +10,7 @@ config ARCH_KEYSTONE
 	select KEYSTONE_TIMER
 	select KEYSTONE2_DMA_COHERENT
 	select MIGHT_HAVE_PCI
+	select OMAP_GPMC
 	select PCI_DOMAINS if PCI
 	select PM_GENERIC_DOMAINS if PM
 	select PINCTRL
-- 
1.7.5.4

